src/components/Timeline/TimelineHours.tsx

```diff
@@ -66,7 +66,9 @@ const HourItem = ({
   theme: ThemeProperties;
 }) => {
   const hourLabelStyle = useAnimatedStyle(() => {
-    return { top: timeIntervalHeight.value * index - 6 + spaceContent };
+    return {
+      top: timeIntervalHeight.value * index - 6 + spaceContent,
```

Comment:
Mixed math needs parens so order of operations is clear

Reply:
This was an unnecessary change that I mistakenly made. I have reverted it, so coming in from the library. I don't think we need to change the maths parenthesis in the library